api/src/main/java/io/strimzi/api/kafka/model/KafkaAuthentication.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2018, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.api.kafka.model;
+
+import io.strimzi.api.kafka.model.type.SslClientAuthType;
+import io.strimzi.crdgenerator.annotations.Description;
+import io.strimzi.crdgenerator.annotations.Example;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import com.fasterxml.jackson.annotation.JsonAnyGetter;
+import com.fasterxml.jackson.annotation.JsonAnySetter;
+import io.sundr.builder.annotations.Buildable;
+
+/**
+ * Configures the broker authorization
+ */
+@Buildable(
+        editableEnabled = false,
+        generateBuilderPackage = true,
+        builderPackage = "io.strimzi.api.kafka.model"
+)
+public class KafkaAuthentication {
+    private SslClientAuthType tlsClientAuthentication;
+    private Map<String, Object> additionalProperties = new HashMap<>(0);
+
+    @Description("Configures TLS Client Authentication on port 9093. Valid values are `required`, `requested` and `none`. When not specified, the default value will be `none`.")
```

Comment:
You can break the long line into multiple `+`-concatenated strings.

You should probably elaborate a little more about what the different value mean.